### ISSUE TITLE
[Feature request] IR camera for markers

### ISSUE THREAD

**Issue by severgun** [ph0b/opentrack-realsense-plugin#2]
Is it possible to use ir camera without laser?

**Comment by ph0b** [comment]
Yes, using [librealsense ](https://github.com/IntelRealSense/librealsense) with some small modifications or the RealSense SDK with that call:
`senseManager->QueryCaptureManager()->QueryDevice()->SetIVCAMLaserPower(0);`
